polars/polars-lazy/polars-plan/src/dsl/function_expr/strings.rs

```diff
@@ -1,11 +1,16 @@
 use std::borrow::Cow;
 
+#[cfg(feature = "regex")]
+use once_cell::sync::Lazy;
 use polars_arrow::utils::CustomIterTools;
 #[cfg(feature = "regex")]
 use regex::{escape, Regex};
 #[cfg(feature = "serde")]
 use serde::{Deserialize, Serialize};
 
+#[cfg(feature = "regex")]
```

Comment:
I think this should be under feature `timezones` and then the `timezones` feature can activate `regex`.